scripts/check-peer-constraints.go

```diff
@@ -0,0 +1,110 @@
+// Copyright (C) 2019 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+// +build ignore
+
+// check-peer-constraints checks that none of the core packages import peers directly.
+package main
+
+import (
+	"flag"
+	"fmt"
+	"os"
+	"strings"
+
+	"golang.org/x/tools/go/packages"
+)
+
+var race = flag.Bool("race", false, "load with race tag")
+var fail = flag.Bool("fail", true, "fail on violation")
+
+func main() {
+	flag.Parse()
+
+	var exitcode int
+
+	peers, err := load(
+		"storj.io/storj/satellite/...",
+		"storj.io/storj/storagenode/...",
+		"storj.io/storj/bootstrap/...",
```

Comment:
Should we add the versioncontrol here as well?
It doesnt have any sub packages yet, but that could happen at any stage

Reply:
added